### DISCUSSION TITLE
Get map type

### DISCUSSION THREAD

**Discussion by cembora** [SupasinTatiyanupanwong/map-kit-android#5] [General]
Is there a way to know which map is in use?. Like MapClient.isHuawei(). If not I think that can be helpful. I need that for custom url for tiles.

**Comment by SupasinTatiyanupanwong** [comment]
That is an interesting idea, indeed. But for now, the only way to achieve that is to do it via reflection.

Note that, Bintray is going to stop accepting packages submission at the end of this month. If you'll find any issues, feels free to fire them all, and I'll try to fix and roll them out. Then, I'll have to find some time to switch to Maven Central.

**Comment by SupasinTatiyanupanwong** [comment]
The library is currently performing API checks at initialization, I'll expose an API for this check at MapKit class instead of MapClient in the upcoming version.

Let me know if there are any use case such that map type should be changable at runtime, I'll consider this in the upcoming major version since it seems to be major change.
